das/daser.go

```diff
@@ -168,6 +169,8 @@ func (d *DASer) sample(ctx context.Context, h *header.ExtendedHeader) error {
 			"square width", len(h.DAH.RowsRoots), "data root", h.DAH.Hash(), "err", err)
 		return err
 	}
+	log.Debugw("sampled header", "height", h.Height, "hash", h.Hash(),
+		"square width", len(h.DAH.RowsRoots), "data root", h.DAH.Hash(), "finished (s)", time.Since(start))
```

Comment:
We might want to move it to the worker loop because metrics introduce the same time tracking, and doing it twice is wasteful.

Reply:
Good point. updated